packages/mds-transactions-api/tests/transaction-operations.spec.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright 2020 City of Los Angeles
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import supertest from 'supertest'
+import { ApiServer } from '@mds-core/mds-api-server'
+import { transactionOperationsGenerator, TransactionServiceClient } from '@mds-core/mds-transactions-service'
+// import { SCOPED_AUTH } from '@mds-core/mds-test-data'
```

Comment:
remove if not needed

Reply:
Reasoning below for why it's commented out at the moment... There's something very odd with importing `mds-test-data` from a Jest test suite within the monorepo that causes an OOM runtime error 😢

Reply:
big oof, but 👍